src/main/java/org/folio/services/validator/InvoiceValidator.java

```diff
@@ -44,22 +45,32 @@
 
   public void validateInvoiceProtectedFields(Invoice invoice, Invoice invoiceFromStorage) {
     if(isPostApproval(invoiceFromStorage)) {
-      Set<String> fields = findChangedProtectedFields(invoice, invoiceFromStorage, InvoiceProtectedFields.getFieldNames());
+      Set<String> fields = findChangedFields(invoice, invoiceFromStorage, InvoiceProtectedFields.getFieldNames());
       verifyThatProtectedFieldsUnchanged(fields);
     }
   }
 
+  public boolean validateInvoiceUnprotectedFields(Invoice invoice, Invoice invoiceFromStorage) {
```

Comment:
Based on code standards this method better to named : isInvoiceUnprotectedFieldsChanged()